src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchFilterTest.java

```diff
@@ -439,9 +451,19 @@ private void testByConfig(String configPath, String inputFile, String[] expected
 
         // run RootModule
         final String path = getPath(inputFile);
-        final List<File> files = Collections.singletonList(
-                new File(path));
-        final int errorCounter = rootModule.process(files);
+        final File file = new File(path);
+        final int errorCounter;
+        final List<File> theFiles;
+        if (file.isDirectory()) {
+            final File[] files = new File(path).listFiles();
+            theFiles = new ArrayList<>();
+            Collections.addAll(theFiles, files);
```

Comment:
can be done easier:
```java
theFiles = Arrays.asList(new File(path).listFiles())
```
(returned list is read-only though)

Reply:
done